config/default.yml

```diff
@@ -989,6 +989,13 @@ Metrics/PerceivedComplexity:
 Lint/AssignmentInCondition:
   AllowSafeAssignment: true
 
+Lint/BlockAlignment:
+  AlignWith: either
```

Comment:
I was trying to be consistent with https://github.com/bbatsov/rubocop/pull/2900/files#diff-e93280b3b31a6438c533a5f3232340d8R1008 and https://github.com/bbatsov/rubocop/pull/2900/files#diff-e93280b3b31a6438c533a5f3232340d8R1020

I'm happy to do either. Which do you want?